src/common/transformations/tests/common_optimizations/align_mixed_fp32_fp16_types_test.cpp

```diff
@@ -36,24 +35,21 @@ TEST_F(TransformationTestsF, align_mixed_fp16_fp32_1) {
         pass::Manager manager;
         manager.register_pass<pass::MarkSugraphsToKeepInMixedPrecision>();
         manager.register_pass<pass::AlignMixedFP32FP16Types>();
-        const precisions_array convert_precision_list{{ov::element::f32, ov::element::f16}};
-        type_to_fuse_map empty_type_to_fuse_map = {};
-        manager.register_pass<pass::ConvertPrecision>(convert_precision_list, empty_type_to_fuse_map, true);
```

Comment:
No need to call `ConvertPrecision` manually. Now `MarkSugraphsToKeepInMixedPrecision` and `AlignMixedFP32FP16Types` are called inside `ConvertPrecision`

Reply:
Done. Added tests for `ConvertPrecision` with flag `skip_precision_sensitive=True`

Reply:
I think we can do both. I added tests for `ConvertPrecision` with flag `skip_precision_sensitive=True`, but i think keeping tests for marking and alignment does also makes sense.

If we accidentally break `AlignMixedFP32FP16Types` or `ConvertPrecision` we will see only red tests for `ConvertPrecision` but for marking they will be green: such info can help us to localize place where we did something wrong. Like the essence of unit-tests - we have several steps and we test each step, not only final one.